### ISSUE TITLE
Write section: What is a systematic review

### ISSUE THREAD

**Issue by vsimko** [biggis-project/UHI-prediction-paper#5]
Summarize what is a systematic review
- For guidelines see issue #4
- A systematic review follows a well-defined method to identify, analyze, synthesize, evaluate, and compare all available literature works relevant to a specific research topic
- A systematic review is an important piece of work since it summarizes existing techniques concerning a research interest, it identifies further research directions, and it provides a framework to position new research activities

**In short**:
 1. Clearly define research questions that the systematic review is expecting to answer
 2. develop a search strategy which includes a paper selection procedure, resources to be searched, and inclusion and exclusion criteria
 3. use the search strategy to identify the relevant research works
 4. analyze, synthesize, and compare the related works to answer the research questions.
